src/EiffelEvents.Net/Exceptions/SchemaNotFoundException.cs

```diff
@@ -0,0 +1,32 @@
+﻿//    Copyright (c) 2021 ICT Cube, doWhile, and Eiffel Community collaborators.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//        http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+using System;
+
+namespace EiffelEvents.Net.Exceptions
+{
+    /// <summary>
+    /// SchemaNotFoundException raised when json schema not found
+    /// </summary>
+    public class SchemaNotFoundException : EiffelException
+    {
+        public SchemaNotFoundException(string message) : base(message)
```

Comment:
I suggest to refactor `SchemaNotFoundException` class to have "Event json schema is not found for {eventName}" and take the event name as a parameter, so a consistent error message be exist.

for example 
```
public SchemaNotFoundException(string eventName) : 
            base($" Event json schema is not found for {eventName}")
```

Reply:
Should probably take version as well then, to be extra clear.